source/reporting/index.html.md.erb

```diff
@@ -91,6 +91,33 @@ have entered when making a payment:
     },
 ```
 
+### Payment service provider (PSP) fees
+
+If you use GOV.UK's Payment Service Provider (PSP), we'll automatically deduct the PSP's transaction fee from each user payment. You can see how much the fee was when you make the API call to get information about a payment.
+
+The JSON response body will contain for example:
+
+```javascript
+{
+  "total_amount": 100,
+  "fee": 1.5,
+  "net_amount": 98.5
+}
```

Comment:
maybe change this example as we denominate all amounts in pence, so there will never be a fractional amount, fee or net

Reply:
Thanks @mrlumbu - I mixed up pounds and pence. I've fixed this now.